cli/checkconfig.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"path/filepath"
+
+	"github.com/johnewart/subcommands"
+
+	. "github.com/yourbase/yb/packages"
+	. "github.com/yourbase/yb/plumbing"
+	. "github.com/yourbase/yb/types"
+	. "github.com/yourbase/yb/workspace"
+)
+
+type CheckConfigCmd struct {
+}
+
+func (*CheckConfigCmd) Name() string     { return "checkconfig" }
+func (*CheckConfigCmd) Synopsis() string { return "Check the config file syntax" }
+func (*CheckConfigCmd) Usage() string {
+	return `checkconfig`
+}
+
+func (b *CheckConfigCmd) SetFlags(f *flag.FlagSet) {
+}
+
+func (b *CheckConfigCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
+	var targetPackage Package
+
+	if PathExists(MANIFEST_FILE) {
+		currentPath, _ := filepath.Abs(".")
+		_, pkgName := filepath.Split(currentPath)
+		pkg, err := LoadPackage(pkgName, currentPath)
+		if err != nil {
+			fmt.Printf("Error loading package '%s': %v\n", pkgName, err)
+			return subcommands.ExitFailure
+		}
+		targetPackage = pkg
+	} else {
+
+		workspace, err := LoadWorkspace()
+
+		if err != nil {
+			fmt.Printf("No package here, and no workspace, nothing to check!")
```

Comment:
Do you think a completely new user of yourbase is going to understand this message? Is there a way we can be more helpful and clear?

Even I don't know what workspace and project mean in this context. How about:

"Could not find .yourbase.yml file. Try running yb in the repository root or writing the YAML config file if it missing. See https://docs.yourbase.io"

Reply:
Good call

Reply:
Actually, this entire else block os for loading the Workspace, not directly the .yourbase.yml file. But I get your point, I'll improve the user messages.